client/src/pages/SignUp.js

```diff
@@ -0,0 +1,17 @@
+import * as React from "react";
+import Navbar from "../components/Navbar";
+import SignUpForm from "../components/SignUpForm";
+import Footer from "../components/Footer";
+
+
+export default function SignUp() {
+    return(
+        <>
+            <section>
+                <Navbar />
+                <SignUpForm />
+            </section>
+            <Footer />
+        </>
+    );
+}
```

Comment:
Hi Maziar,
We'll create pages with routes as in App.js.
This component will not be needed.